src/runtime_src/core/tools/common/xball.py

```diff
@@ -0,0 +1,83 @@
+import json
```

Comment:
Can we move this file under src/pyhton/tools?

Reply:
I ended up keeping the python code in the bash script as it originally was. Adding the python directory and file under yocto was a bit much just for this file